src/app/ReadClient.cpp

```diff
@@ -624,6 +624,8 @@ CHIP_ERROR ReadClient::SendSubscribeRequest(ReadPrepareParams & aReadPreparePara
         SuccessOrExit(err);
     }
 
+    VerifyOrExit(aReadPrepareParams.mMinIntervalFloorSeconds < aReadPrepareParams.mMaxIntervalCeilingSeconds,
```

Comment:
The spec allows these to be equal, as far as I can tell.  Why the strict inequality here?

Reply:
per chat with @mrjerryjohns , min would not equal to max

Reply:
What does this chat have to do with anything?  The spec does not say they can't be equal.  It requires some values to be >= the min and <= the max, which can work perfectly well if they are equal.

If you think the should never be equal, the very least that's needed here is an SDK feedback spec issue, not just randomly implementing whatever we want that violates the spec without bothering to tell anyone.